js/main.js

```diff
@@ -98,3 +149,18 @@ document.addEventListener('DOMContentLoaded', function (event) {
     swapView('profile');
   }
 });
+var $modal = document.querySelector('#modal');
+
+document.addEventListener('click', function (event) {
+  if (event.target.tagName === 'A' && event.target.className !== 'header-profile') {
```

Comment:
Based on this, you could never actually get to the `"profile"` view. This is checking if the user clicked an anchor tag with the class `header-profile`, which is the only anchor tag you currently have. That means if they click on "Profile", they're going to go to edit profile view. You should be checking where the user wants to go based on the value of the `data-view` attribute.

Additionally, the check for if the tag clicked is an `a` tag should be a separate conditional, where if the element is not a anchor tag, you exit the function immediately. That way, the conditional that follows it can simply be what we want to do, because if the code got past the first conditional, that means the user clicked on an `a` tag.